lib/service.js

```diff
@@ -213,6 +218,52 @@ class Service extends EventEmitter {
     eventClass.convert(this, rawEvent, cb);
   }
 
+  /**
+   * Get Targets For EC2 Task
+   *
+   * Finds the load balancer targets for the EC2 task based on container instances.
+   *
+   * @param {object} task The task we are resolving targets for.
+   * @param {object} container The specific container we are looking for targets associated with.
+   * @return {array} A list of targets
+   */
+  _getTargetsForEC2Task(task, container) {
+    const containerInstance = this.getContainerInstance(task.containerInstanceArn);
+
+    return container.networkBindings.map((binding) => {
+      return this.getTarget(containerInstance.ec2InstanceId, binding.hostPort);
+    }).filter((target) => !!target);
+  }
```

Comment:
Random Aside: If a task is running in awsvpc mode on EC2, I assume this still works?

Reply:
I have no idea. I'd prefer to make that change (if we deem it necessary) later on as a separate PR.